src/reducers/index.js

```diff
@@ -2,11 +2,14 @@ import { combineReducers } from 'redux';
 import auth from './auth';
 import signinuser from './signin';
 import signup from './signup';
+import { articleComments, newComment } from './comments';
 
 const rootReducer = combineReducers({
   auth,
   signinuser,
   signup,
+  articleComments,
```

Comment:
The initial state has the `articleComments` and `newComment` nested inside the `comments` object. But the rootReducer is not implementing the same structure. Please fix this.

Reply:
Thanks. I've fixed it.